client/signup/steps/rebrand-cities-welcome/index.jsx

```diff
@@ -54,9 +60,9 @@ class RebrandCitiesWelcomeStep extends Component {
 
 		return (
 			<div className="rebrand-cities-welcome">
-				<div className="rebrand-cities-welcome__illustration-wrapper">
-					<div className="rebrand-cities-welcome__illustration" />
-				</div>
+				<Card className="rebrand-cities-welcome__illustration-wrapper">
```

Comment:
Hi! Why couldn't we retain the CSS definition for the contained image? 

I see Card offers the white background and other styles, but the usage here doesn't look to match [Card's intended purpose](https://github.com/Automattic/wp-calypso/tree/master/packages/components/src/card).

Also, there might be side effects if, for any reason, Card's styles change.

Reply:
I used the Card option because that's what was used for most of the changes in #30004, such as the Reader page: https://wordpress.com/start/reader. A lot of images have a white background, though I've not checked if they're using the Card component yet. 

<img width="825" alt="51397817-05aa8a00-1b10-11e9-9431-f6a10d87f3ae" src="https://user-images.githubusercontent.com/43215253/70415657-0f551d00-1a55-11ea-99bc-6859202f2d62.png">

But I think it was mainly an oversight that the image wasn't wrapped in one (maybe?), though I'm happy to use an alternative. :)

I removed the image styles because the image was using `background-image`. I don't think there was any advantage in this? When just using `<img>`, those styles aren't necessary, and actually break the image a bit with the increased padding.

Reply:
> When just using <img>, those styles aren't necessary, and actually break the image a bit with the increased padding.

No worries. I was, not very clearly, suggesting that we could keep the style definition instead of relying on the CSS in `<Card />`.

If `<Card />` is being used around the shop however, I guess it's harmless up until the point that we need something custom for that page.

Cheers!

Reply:
I added a bit of styling so that the Card component is no longer necessary. :)